tests/test_torch_core.py

```diff
@@ -101,6 +103,28 @@ def test_np_address():
     add=np_address(a)
     assert isinstance(add, int)
     
+def test_to_data():
+    this_tests(to_data)    
+    path = untar_data(URLs.MNIST_SAMPLE)
+    data1 = ImageDataBunch.from_folder(path)
+    ys1 = list(data1.y)
+    a=([1,2,3],[3,6,6])
+    b=([4,5,6],[4,7,7])
+    data2 = torch.tensor([a,b])
+    ys2= list(data2[0])
+    assert isinstance(data1, fastai.vision.data.ImageDataBunch)
+    assert isinstance(data1.y, ItemList)
+    assert isinstance(ys1, list)
+    assert isinstance(ys1[0], (ItemBase or fastai.core.Category))
```

Comment:
Why does Category not work here?

Reply:
Finally got time to look at it. 
Correct me if I'm wrong, I think you mean it is preferred to use:
assert isinstance(ys1[0], (Category))

instead of:
`assert isinstance(ys1[0], (ItemBase or fastai.core.Category))`

Reply:
Yes